aws-api-appsync/src/main/java/com/amplifyframework/core/model/types/JavaFieldType.java

```diff
@@ -46,6 +46,11 @@
      */
     FLOAT(Float.class),
```

Comment:
Could we get rid of the `FLOAT` type at some point in the future?  We can't now, since the codegen'd models all have `Float` objects, but if we change codegen to generate Double objects, then we could, right?

Reply:
Yes! Down the road, when the majority of folks are using the new modelgen, we could drop the Floats. Truthfully I suspect we could do it now, if we showed people how to update their models. But, it would technically be a breaking change.